### PR TITLE
Add Sonance brand

### PR DESCRIPTION
**Pull request by bmillemathias**
over-pass link https://overpass-turbo.eu/s/1ndj
I did not put a name tag on purpose because Sonance is a netword of independant shops and I'm not sure all shops are named Sonance

### CODE REVIEW

**Comment by UKChris-osm** [comment]
I'm not sure if using the network tag here would be needed as the brand tag would be the main tag here, although from the Overpass link I do see network is used a lot.

If you're unsure about the name, you can add "preserveTags["^name"]; under the locationSet and this will only add a name if there isn't one already assigned (like a default). This means if someone has already added a name, and just wants to add the brand data, the name will not be overwritten.

File: data/brands/shop/hearing_aids.json [hunk 1]
```diff
@@ -180,6 +180,17 @@
...
       }
     },
+    {
+      "displayName": "Sonance Audition",
+      "id": "sonanceaudition-28a4ee",
+      "locationSet": {"include": ["fr"]},
+      "tags": {
+        "brand": "Sonance Audition",
+        "brand:wikidata": "Q114932560",
+        "network": "Sonance Audition",
```
Comment (LaoshuBaby) [inline data/brands/shop/hearing_aids.json:190]:
```suggestion
        "name": "Sonance Audition",
```

**Review by LaoshuBaby — CHANGES_REQUESTED** [review]

File: data/brands/shop/hearing_aids.json (hunk 1 above)
Comment (bmillemathias) [reply data/brands/shop/hearing_aids.json:190]: as I said not all shops have the same namethat why I used network key

**Comment by bmillemathias** [comment]
> I'm not sure if using the network tag here would be needed as the brand tag would be the main tag here, although from the Overpass link I do see network is used a lot.

THis is because of me, I updated the shop before submitting the MR

> 
> If you're unsure about the name, you can add "preserveTags["^name"]; under the locationSet and this will only add a name if there isn't one already assigned (like a default). This means if someone has already added a name, and just wants to add the brand data, the name will not be overwritten.

Thanks for the suggestion, I´ll do that

**Comment by UKChris-osm** [comment]
What I would recommend in this PR is adding the name tag as well, as this will act as the default for any new addition to OSM. But, by having `preserveTags["^name"]` you are protecting the name if it's been changed, so iD / NSI won't attempt to overwrite it.

**Comment by bhousel** [comment]
Thanks @bmillemathias  👍 

> What I would recommend in this PR is adding the name tag as well, as this will act as the default for any new addition to OSM.

This is a good suggestion, I added this as a followup commit in 2b55955

**Comment by bmillemathias** [comment]
Thanks !

**Pull request merged**